engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -179,7 +179,7 @@ public void stop()
     yarnClient.stop();
   }
 
-  public static LinkedHashSet<String> findJars(LogicalPlan dag, Class<?>[] defaultClasses)
+  public LinkedHashSet<String> findJars(Class<?>[] defaultClasses)
```

Comment:
Logically the method findJars() is an internal method of the class  StramClient. The PR implementation has to use an extra private property of the class StramClient: Configuration conf. So I think it is much better to change the method to dynamic and exclude dag and configuration from the method signature.